CMakeLists.txt

```diff
@@ -1,6 +1,12 @@
 cmake_minimum_required (VERSION 2.8.12)
 project (PIO C Fortran)
 
+# The project version number.
+set(VERSION_MAJOR   2   CACHE STRING "Project major version number.")
+set(VERSION_MINOR   3   CACHE STRING "Project minor version number.")
+set(VERSION_PATCH   8   CACHE STRING "Project patch version number.")
+mark_as_advanced(VERSION_MAJOR VERSION_MINOR VERSION_PATCH)
+
```

Comment:
The current version (tag) in git is 2_0_26 (there is some legacy reason that i've long forgotten for using _ instead of . we can probably change that)  Is there a way to automatically have that git tag inserted here?